packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -250,9 +250,9 @@ export async function fluidFetchMessages(documentService?: IDocumentService, sav
         return;
     }
 
-    const files = !saveDir ?
-        undefined :
-        fs.readdirSync(saveDir)
+    const files = saveDir === undefined
+        ? undefined
+        : fs.readdirSync(saveDir)
```

Comment:
Not really related to this PR, but do you know why we're using the synchronous version of this in an async function? Should we file a bug for this?

Reply:
#10659